examples/config/cosmodc2_config.yml

```diff
@@ -16,6 +16,22 @@ TXCosmoDC2Mock:
     flip_g2: True # to match metacal
 
 
+TXMainMaps:
+    chunk_rows:  100000
+    pixelization: healpix
+    nside: 512
+    sparse: True
+
+TXAuxiliaryMaps:
+    chunk_rows:  100000
```

Comment:
I wonder if we want to set this a variable within the config file since it's repeated so many times!

Reply:
Yeah, that's a good idea. You can use the section "global" for configuration that (if needed) goes to every stage, and I've pushed changes to do that.